docs/user/check-ref/entities_wrapper.rst

```diff
@@ -0,0 +1,84 @@
+Entities
+--------
+
+Provides access to ZMON entities.
+
+.. py:function:: entities(service_url, infrastructure_account, verify=True, oauth2=False)
+
+    ::
+
+    Initialize entities wrapper.
+
+    :param service_url: Entities service url.
+    :type service_url: str
+
+    :param infrastructure_account: Infrastructure account used to filter entities.
+    :type infrastructure_account: str
+
+    :param verify: Verify SSL connection. Default is ``True``.
+    :type username: bool
+
+    :param oauth2: Use OAUTH for authentication. Default is ``False``.
+    :type oauth2: bool
+
+
+Methods of Entities
+^^^^^^^^^^^^^^^^^^^
+
+.. py:function:: search_local(**kwargs)
+
+    ::
+
+    Search entities in local infrastructure account. If `infrastructure_account` is not supplied in kwargs, then the one supplied upon intialization will be used.
```

Comment:
Rephrase for "initialization".

Search entities "local" to your filtered entities by using the same infrastructure_account as a default filter.